packages/plugins/users-permissions/admin/src/pages/Roles/EditPage/tests/index.test.js

```diff
@@ -967,4 +967,18 @@ describe('Admin | containers | RoleEditPage', () => {
     const errorMessages = await getAllByText(/invalid value/i);
     errorMessages.forEach(errorMessage => expect(errorMessage).toBeInTheDocument());
   });
+
+  it('can toggle the permissions accordions', async () => {
+    // Create app and wait for loading
+    const { getByLabelText, queryByText, getByTestId, getByText } = makeAndRenderApp();
+    const loader = getByTestId('loader');
+    await waitForElementToBeRemoved(loader).catch(e => console.error(e));
```

Comment:
Why do you need the catch? It should throw if it fails I think :)

Reply:
I think I blindly copied some code from the testing-library's website, but didn't realize I actually wanted it to throw 😅 

<img width="829" alt="Screenshot 2021-09-20 at 10 48 54" src="https://user-images.githubusercontent.com/8087692/133976962-76bfebfa-6475-4585-8d14-ca6dd6e75b75.png">